tests/conftest.py

```diff
@@ -519,5 +584,22 @@ async def swarm7(request):
     run(["make", "kill-anvil", f"port={anvil_port}"], cwd=PWD.parent, check=True)
 
 
+@pytest.fixture(autouse=True)
+async def teardown(swarm7: dict[str, Node]):
+    yield
+
+    try:
+        await asyncio.gather(*[node.api.reset_tickets_statistics() for node in swarm7.values()])
+    except Exception as e:
+        logging.error(f"Error resetting tickets statistics in teardown: {e}")
+        raise e
+
+    try:
+        await asyncio.gather(*[node.api.messages_pop_all(0) for node in swarm7.values()])
```

Comment:
I think it might require `messages_pop_all(None)` instead to clear all tags. Could you pls double-check ?